### PR TITLE
Add keyboard shortcut for toggling annotations

### LINKED ISSUE #3900: Add annotation toggle hotkey
### What article on docs.github.com is affected?
https://docs.github.com/en/github/getting-started-with-github/keyboard-shortcuts#source-code-browsing

### What part(s) of the article would you like to see updated?
Github now allows a shortcut of `a` on to show and hide all annotations on a file diff.

### PR DESCRIPTION
**Pull request by aaron-tdouble**
### Why:

The Special Projects team recently released a feature that allows a user the ability to toggle all diff annotations via the `a` hotkey across all files, similar to the way that `i` does for comments.

This change adds the hotkey to the docs to increase exposure of this new ability.

Issue:  https://github.com/github/special-projects/issues/89
Merged github PR: https://github.com/github/github/pull/171199

**Closes [issue link]** https://github.com/github/docs/issues/3900

### What's being changed:

Adding this hotkey entry:

Keyboard shortcuts - GitHub Docs 2021-02-22 14-35-43

### Check off the following:
- [X] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [X] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by Devilishhh — APPROVED** [review]

**Comment by aaron-tdouble** [comment]
@Devilishhh thanks for the approval.  The PR is still showing as needing at least one approval which is weird (since you obviously approved).  What do I need to do in order to allow this to be merged?

**Review by Devilishhh — CHANGES_REQUESTED** [review]
This needs to be merged

**Comment by Devilishhh** [comment]
> @Devilishhh thanks for the approval.  The PR is still showing as needing at least one approval which is weird (since you obviously approved).  What do I need to do in order to allow this to be merged?

Needs to be merged by someone that is authorised to do so

**Comment by janiceilene** [comment]
@aaron-tdouble Thanks so much for opening a PR and an issue! I'll get this triaged for writer review ⚡ 

In the future, could you open an issue in [docs content](https://github.com/github/docs-content/issues/new?assignees=&labels=%5B%22Improve+existing+docs%22%5D&template=improve-existing-docs.md&title=)? That'll give writers an opportunity to weigh in on what changes will be needed (that plan is outlined in https://github.com/github/docs-team/issues/340#issuecomment-769974718).

**Review by janiceilene — APPROVED** [review]
This change looks great!

File: content/github/getting-started-with-github/keyboard-shortcuts.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ For more keyboard shortcuts, see the [CodeMirror documentation](https://codemirr
 |<kbd>w</kbd> | Switch to a new branch or tag
 |<kbd>y</kbd> | Expand a URL to its canonical form. For more information, see "[Getting permanent links to files](/articles/getting-permanent-links-to-files)."
 |<kbd>i</kbd> | Show or hide comments on diffs. For more information, see "[Commenting on the diff of a pull request](/articles/commenting-on-the-diff-of-a-pull-request)."
+|<kbd>a</kbd> | Show or hide annotations on diffs
```
Comment (nenehulc) [inline content/github/getting-started-with-github/keyboard-shortcuts.md:67]: ``

**Review by nenehulc — CHANGES_REQUESTED** [review]

**Comment by nenehulc** [comment]
Wooow

**Review by Devilishhh — COMMENTED** [review]
All good

**Review by Irahaze5 — APPROVED** [review]

**Review by Irahaze5 — APPROVED** [review]

**Pull request merged**